app/src/main/res/layout/list_item_plant.xml

```diff
@@ -20,16 +20,14 @@
     xmlns:tools="http://schemas.android.com/tools">
 
     <data>
-
         <variable
-            name="vm"
-            type="com.google.samples.apps.sunflower.adapters.PlantAdapter.ItemViewModel"/>
+            name="plant"
+            type="com.google.samples.apps.sunflower.data.Plant"/>
```

Comment:
I'm not so strictly against direct connection between domain and UI, however, I would only suggest using a view-model to a capsulate domain in order to isolate UI and logical with the view-model.

Reply:
In the 2018 version of the Google I/O Android App we used this pattern for RecyclerView list items.  I think a ViewModel is a bit heavyweight for this use case.